src/test/java/org/apache/accumulo/access/AccessEvaluatorTest.java

```diff
@@ -89,11 +89,10 @@ public void runTestCases() throws IOException {
         evaluator = AccessEvaluator.builder().authorizations(testSet.auths[0]).build();
         runTestCases(testSet, evaluator);
 
-        evaluator = AccessEvaluator.builder().authorizations(testSet.auths[0]).cacheSize(1).build();
+        evaluator = AccessEvaluator.builder().authorizations(testSet.auths[0]).build();
```

Comment:
These two tests runs can be removed because they are now repeating the same tests.

Reply:
Nice catch. Fixed in ea431d5